aws-athena/views/default-vw_pin_universe.sql

```diff
@@ -1,4 +1,29 @@
 -- Source of truth view for PIN location
+
+-- This CTE ensures the most recent year of pardat data is joined to spatial
+-- data regardless of timing. This is necessary since spatial data releases late
+-- in a calendar year while the CCAO's universe of PINs is rolled-over much
+-- sooner.
```

Comment:
**suggestion:** Can you add a small comment diagram that shows the missingness problem? I feel like the timing/missingness here is really confusing if you're not intimately familiar with the data release schedule.